src/components/tinyangel/TinyAngel.tsx

```diff
@@ -0,0 +1,244 @@
+                                </Typography>
+                                <Slider 
+                                onChange={(e: any) => setNativeWalletState({ tinyValue: +e.target.value })}
+                                color="primary"
+                                min={0.5} 
+                                max={5}
+                                step={0.1}
+                                valueLabelDisplay="auto"/>
+                            </Stack>
+
+                            { nativeWalletState.tinyBalances.length === 0 &&
+                                <Typography>No Tiny Balances</Typography>
+                            }
+
+                            {nativeWalletState.tinyBalances.length > 0 && 
+                                <Grid 
+                                container 
+                                gap="10px">
+                                    {nativeWalletState.tinyBalances.map(balance => (
+                                        <Grid
+                                        border="1px solid white"
+                                        borderRadius="10px">
+                                            <Typography variant="h6" sx={{margin: '10px'}}>
+                                                {visualDenomName.get(balance.denom)}
+                                            </Typography>
+                                            <Typography variant="h6" sx={{margin: '10px'}}>
+                                                {toTerraAmount(balance.amount).toFixed(6)}
+                                            </Typography>
+                                        </Grid>
+                                    ))}
+                                </Grid>
+                            }
+                            <Button variant="contained" onClick={ onDonate }>Donate</Button>
+                        </Stack>
+                    </Paper>
+                </Grid>
+                <Grid item xs={12} sm={6}>
+                    <Paper elevation={3} style={{height: 'fit-content'}}>
+                        <Stack spacing={1} sx={{padding: '30px'}}>
+                            <Typography 
+                            variant="h5" 
+                            width="100%" 
+                            textAlign="center">
+                                Donate Tiny Rewards from Staking
+                            </Typography>
+
+                            <Stack
+                            padding="15px 0">
+                                <Typography>
+                                    Tiny Reward Upper Limit (in UST)
+                                </Typography>
+                                <Slider 
+                                onChange={(e: any) => setRewardState({ tinyReward: +e.target.value })}
+                                color="primary"
+                                min={0.000001} 
+                                max={1}
```

Comment:
Same change as above but make this between 0.5 and 5 as well

Reply:
done